activesupport/lib/active_support/jsonapi.rb

```diff
@@ -0,0 +1 @@
+require 'active_support/jsonapi/decoding'
```

Comment:
agreed.  just wondering what it means for it to be missing, and if there should maybe be a no-op version to assist in deciding if this is the right api